### PR TITLE
New option + checks for add-parents.

### PR DESCRIPTION
**Pull request by clemens-tolboom**
This solve https://github.com/fhaculty/graph-uml/issues/2 and add option to just render given classes by 'add-parents'.

To render only the classes (without adding relations and/or parent class interfaces try this.

``` php

$graph = new Fhaculty\Graph\Graph();
$builder = new Fhaculty\Graph\Uml\ClassDiagramBuilder($graph);
$builder->setOption('add-parents', FALSE);

$builder->createVertexClass('Fhaculty\Graph\Edge\Base');
$builder->createVertexClass('Fhaculty\Graph\Edge\Directed');

$graphviz = new Fhaculty\Graph\GraphViz($graph);
$graphviz->setFormat('svg');
$graphviz->display();
```

### CODE REVIEW

File: src/Fhaculty/Graph/Uml/ClassDiagramBuilder.php [hunk 1]
```diff
@@ -40,6 +40,8 @@ class ClassDiagramBuilder
         'show-protected' => true,
         // whether to show class constants as readonly static variables (or just omit them completely)
         'show-constants' => true,
+        // whether to show add parent classes or interfaces
```
Comment (clue) [inline src/Fhaculty/Graph/Uml/ClassDiagramBuilder.php:43]: Typo "to show add" :)

File: src/Fhaculty/Graph/Uml/ClassDiagramBuilder.php [hunk 2]
```diff
@@ -76,25 +78,29 @@ public function createVertexClass($class)
         } else {
             $reflection = new ReflectionClass($class);
         }
+        if ($this->graph->hasVertex($class)) {
+            return;
```
Comment (clue) [inline src/Fhaculty/Graph/Uml/ClassDiagramBuilder.php]: Shouldn't be part of this PR? In my opinion it's up for debate in a separate ticket?

**Comment by clue** [comment]
Could you please separate the unit tests from this ticket? Duplicate of #8 ?

Besides my above remarks, this looks good to me. Thanks :+1:

**Comment by clemens-tolboom** [comment]
I'm not sure what to do with https://github.com/clemens-tolboom/graph-uml/commit/2f5b08c2e75b2b68c02c12cbf7e758ce12a2a902 

``` php
+        if ($this->graph->hasVertex($class)) {
+            return;
+        }
```

so made it verbatim for now.

**Comment by clue** [comment]
Very much appreciated, the changes look good to me.

Anything you'd want to sort out before this can be merged?

**Comment by clemens-tolboom** [comment]
The code looks OK to me so please merge :)

**Comment by clemens-tolboom** [comment]
Thanks
